code-challenges.test.js

```diff
@@ -14,39 +14,167 @@
+
+    it("should return an array of words containing 'a'", () => {
+        const letterA = "a"
+        expect(wordsWithLetter(fruitArray, letterA)).toEqual(["Mango", "Apricot", "Peach"])
+    })// Expected output: ["Mango", "Apricot", "Peach"]
+    it("should return an array of words containing 'e'", () => {
+        const letterE = "e"
+        expect(wordsWithLetter(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])// Expected output: ["Cherry", "Blueberry", "Peach"]
+    })
+
+})
+
+
 
-const letterA = "a"
-// Expected output: ["Mango", "Apricot", "Peach"]
-const letterE = "e"
-// Expected output: ["Cherry", "Blueberry", "Peach"]
 
 // b) Create the function that makes the test pass.
+// Psuedo code
+// declare a function
+// use the words array and the letter as your paramaters
+// iterate through the array checking for the string that matches your string
+// if it matches return that word in youre new array 
+// declare a new function thatll be filtered words
+// use .filter method to make a new array 
+// use .tolowercase to include capital letters
+// use .includes to check if the word has the specified letter
+// return filteredWords
+
+const wordsWithLetter = (wordsArray, letter) => {
+    const filteredWords = wordsArray.filter(word => word.toLowerCase().includes(letter.toLowerCase()))
+    return filteredWords 
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       5 passed, 5 total
+// Snapshots:   0 total
+// Time:        0.291 s, estimated 1 s
+// Ran all test suites.
+
+
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a "full house". A full house is exactly one pair and one three of a kind.
 
 // a) Create a test with expect statements using the variable provided.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
-const hand4 = [7, 2, 7, 2, 7]
-// Expected output: true
+describe("fullHouse", () => {
+    const hand1 = [5, 5, 5, 3, 3]
+    // Expected output: true
+    const hand2 = [5, 5, 3, 3, 4]
+    // Expected output: false
+    const hand3 = [5, 5, 5, 5, 4]
+    // Expected output: false
+    const hand4 = [7, 2, 7, 2, 7]
+    // Expected output: true
+    it("should return true for [5, 5, 5, 3, 3]", () => {
+        expect(isFullHouse([5, 5, 5, 3, 3])).toBe("Hand is a full house")
+      })
+    
+      it("should return false for [5, 5, 3, 3, 4]", () => {
+        expect(isFullHouse([5, 5, 3, 3, 4])).toBe("Hand is not a full house")
+      })
+    
+      it("should return false for [5, 5, 5, 5, 4]", () => {
+        expect(isFullHouse([5, 5, 5, 5, 4])).toBe("Hand is not a full house")
+      })
+    
+      it("should return true for [7, 2, 7, 2, 7]", () => {
+        expect(isFullHouse([7, 2, 7, 2, 7])).toBe("Hand is a full house")
+      })
+
+})
```

Comment:
You should use the variables within the function call in your expect statement, otherwise there's no point to even have them there. Also, make sure you are looking at the expected output. The prompt is asking for Boolean values, not a string.